pkg/aws_config_server/webserver.go

```diff
@@ -52,23 +52,23 @@ func requireAuthentication(next httprouter.Handle, verifier oidcVerifier) httpro
 		authHeader := r.Header.Get("Authorization")
 		ctx := r.Context()
 		if len(authHeader) <= 0 {
-			logrus.Warn("error: No Authorization header found.")
+			logrus.Error("error: No Authorization header found.")
```

Comment:
```suggestion
			logrus.Debugf("error: No Authorization header found.")
```
Otherwise anyone that pings this service will generate a sentry error/log message